pkg/helper/versions/helpers.go

```diff
@@ -34,14 +34,12 @@ func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP
 		if isSTS && !ocm.HasSTSSupport(v.RawID(), v.ChannelGroup()) {
 			continue
 		}
-		if filterHostedCP {
```

Comment:
This block looks valid if you change the "filterHostedCP" in the caller.

I guess if you don't want versions for HCP, you wouldn't be required to check if they have hosted cp support.